fastlane-env-reminder/src/main.ts

```diff
@@ -26,16 +28,16 @@ export async function run() {
       console.log('`fastlane env` was already provided, exiting');
       return;
     }
-    
+
     if (issueBody.includes('### Feature Request')) {
       console.log('The issue is a feature request, exiting');
       return;
     }
 
     const issueMessage = core.getInput('issue-message');
     const repoToken = core.getInput('repo-token', {required: true});
-    const issue: {owner: string, repo: string, number: number} = context.issue;
-    const client: github.GitHub = new github.GitHub(repoToken)
+    const issue: {owner: string; repo: string; number: number} = context.issue;
```

Comment:
this doesn't look right initially - but my TS is... non existant.

Reply:
Yes, it takes a bit time to get used to TS syntax. 

If you look at the source code of `context.ts`: 

https://github.com/actions/toolkit/blob/5c894298f2f24441718eb049215aa6decd507193/packages/github/src/context.ts#L45-L52

you see that the issue json object is 1:1 with ours 👍